### PR TITLE
T583865: dxNumberBox works incorrect with different positive and negative parts in format

### PR DESCRIPTION
**Pull request by mrjohn0011**

### CODE REVIEW

**Comment by mrjohn0011** [comment]
It does not work with intl because intl parser does not support different positive and negative formatting. @vconst , I think we should write this in our documentation

**Comment by mrjohn0011** [comment]
It has been decided to fix incorrect behavior on intl level too in a separate issue

File: testing/tests/DevExpress.localization/localization.base.tests.js [hunk 1]
```diff
@@ -653,8 +654,13 @@ QUnit.test("input attr 'type' was not localized (Q588810)", function(assert) {
...
 QUnit.module("Localization number");
 
+QUnit.test("parse different positive and negative parts", function(assert) {
+    assert.equal(numberLocalization.parse("(10)", "#0;(#0)"), -10);
+    assert.equal(numberLocalization.parse("-10"), -10);
+    assert.equal(numberLocalization.parse("-10", "#0;(#0)"), -10);
+});
+
 QUnit.test("format: base", function(assert) {
-    //assert.equal(numberLocalization.format(1.2), "1.2");
```
Comment (vconst) [inline testing/tests/DevExpress.localization/localization.base.tests.js:657]: Why is this test case removed?

File: testing/tests/DevExpress.localization/localization.base.tests.js [hunk 2]
```diff
@@ -802,7 +808,6 @@ QUnit.test("format as LDML pattern with custom separators", function(assert) {
 QUnit.module("Localization currency");
 
 QUnit.test("format: base", function(assert) {
-    //assert.equal(currencyLocalization.format(1.2), "$1.2");
```
Comment (vconst) [inline testing/tests/DevExpress.localization/localization.base.tests.js:805]: Why is this test case removed?

**Review by vconst — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.localization/localization.base.tests.js (hunk 2 above)
Comment (mrjohn0011) [reply testing/tests/DevExpress.localization/localization.base.tests.js:805]: I have just remove commented strings from the code

File: testing/tests/DevExpress.localization/localization.base.tests.js (hunk 1 above)
Comment (mrjohn0011) [reply testing/tests/DevExpress.localization/localization.base.tests.js:657]: I have just remove commented strings from the code

**Review by vconst — APPROVED** [review]

**Pull request merged**